src/Entity/FormSubmission.php

```diff
@@ -188,11 +188,16 @@ public function getName(): string
         return $this->name;
     }
 
-    public function getLabel(): string
+    public function getLabel(): ?string
     {
         return $this->label;
     }
 
+    public function getDeadlineDate(): ?\DateTime
+    {
+        return $this->deadlineDate;
+    }
+
```

Comment:
How do you handle the conversion to a date?

Reply:
In SubmissionController::indexAction I mark it as a DateTime property
`$deadlineColumn->setDateTimeProperty(true);`
Then I check this in twig like this
```
{%  if column.dateTimeProperty == true %}
{{ value|date(date_time_format) }}
```

Reply:
Allright, but the field in the DB is a string not a datetime